src/json-patch-duplex.ts

```diff
@@ -524,6 +525,65 @@ module jsonpatch {
     return true;
   }
 
+  
+  /**
+     * Apply a single json-patch on an object tree
+     * Returns the result object.
+     */
+  export function applyPatch<T>(tree: T, patch: Patch<any>): T;
+  export function applyPatch<T>(tree: T, patch: Patch<any>, validate: boolean, touchOriginalTree: boolean): T;
+  export function applyPatch<T>(tree: T, patch: Patch<any>, validate = false, touchOriginalTree = true): T {
```

Comment:
If you want to keep it, the signature needs to be updated to make `touchOriginalTree` optional